docs/source/catalog.rst

```diff
@@ -256,7 +255,7 @@ instantiation.
         args: {}
 
 These alternative plugins can also be given data-source specific names, allowing the
-user to choose at load time with `driver=` as a parameter. Additional arguments may also
+user to choose at load time with `plugin=` as a parameter. Additional arguments may also
```

Comment:
This is the code being referenced: https://github.com/intake/intake/blob/master/intake/catalog/local.py#L146 , where indeed the parameter is called "driver". I believe it would be better to change the docstring to refer to drivers only (which happen to be a type of plugin). 

Have you used `plugin=` to any effect? Perhaps that might exist elsewhere.